src/System.Net.Http/tests/FunctionalTests/HPackTest.cs

```diff
@@ -51,14 +51,14 @@ public async Task HPack_HeaderEncoding(string headerName, string expectedValue,
                 },
                 async server =>
                 {
-                    await server.EstablishConnectionAsync();
-                    (int streamId, HttpRequestData requestData) = await server.ReadAndParseRequestHeaderAsync();
+                    Http2LoopbackConnection connection = await server.EstablishConnectionAsync();
```

Comment:
If we're changing this code anyway, can it just be changed to use HandleRequestAsync?

Reply:
I'll leave this out of this PR since it's already merge conflict prone enough